File: Exercises/05-age-refactor.py
age = int(input('what is you age?'))
if age > 0 and age < 18:
	print('too young to work')
elif age > 65 and age < 125:
	print('past retirement age')
elif age >= 18 and age <= 65:
	print('working age')
else:
    print('invalid age')

File: Exercises/05-age-refactor.js
var age = prompt('what is you age?');
if(age > 0 && age < 18){
	console.log('too young to work');
} else if(age > 65 && age < 125) {
	console.log('past retirement age');
} else if (age >= 18 && age <= 65){
	console.log('working age');
} else console.log('invalid age');